### PR TITLE
code reformatting

### PR DESCRIPTION
**Pull request by A4K-Official**
@host505 @nazegnl Here's the PR mind taking a look? everything looks good to me but just wanted to make sure :)

### CODE REVIEW

File: lib/openscrapers/modules/client2.py [hunk 1]
```diff
@@ -468,7 +483,8 @@ def get(self, result):
             s = re.sub(r'\n', '', s)
             s = re.sub(r'document\.cookie', 'cookie', s)
             cookie = ''
-            exec s
+            exec
```
Comment (nazegnl) [inline lib/openscrapers/modules/client2.py]: exec(s)

File: lib/openscrapers/modules/log_utils.py [hunk 2]
```diff
@@ -38,8 +38,10 @@ def log(msg, level=LOGNOTICE):
     except:
         return
     if xbmc:
-        print DEBUGPREFIX + ' Debug Enabled?: ' + str(debug_enabled)
-        print DEBUGPREFIX + ' Debug Log?: ' + str(debug_log)
+        print
```
Comment (nazegnl) [inline lib/openscrapers/modules/log_utils.py]: print(.......)

File: lib/openscrapers/modules/log_utils.py [hunk 3]
```diff
@@ -38,8 +38,10 @@ def log(msg, level=LOGNOTICE):
...
+        DEBUGPREFIX + ' Debug Enabled?: ' + str(debug_enabled)
+        print
```
Comment (nazegnl) [inline lib/openscrapers/modules/log_utils.py]: print(.......)

File: lib/openscrapers/modules/log_utils.py [hunk 4]
```diff
@@ -52,9 +54,10 @@ def log(msg, level=LOGNOTICE):
                 f.close()
             with open(log_file, 'a') as f:
                 line = '[%s %s] %s: %s' % (datetime.now().date(), str(datetime.now().time())[:8], DEBUGPREFIX, msg)
-                f.write(line.rstrip('\r\n')+'\n')
+                f.write(line.rstrip('\r\n') + '\n')
         else:
-            print '%s: %s' % (DEBUGPREFIX, msg)
+            print
```
Comment (nazegnl) [inline lib/openscrapers/modules/log_utils.py]: print(.......)

File: lib/openscrapers/modules/regex.py [hunk 5]
```diff
@@ -484,177 +496,188 @@ def unwise_func( w, i, s, e):
...
     e = ''
-    d = ''#32823
+    d = ''  # 32823
 
-    #sUnpacked = str(__unpack(p, a, c, k, e, d))
-    sUnpacked1 = str(__unpack(p1, a1, c1, k1, e, d,iteration))
+    # sUnpacked = str(__unpack(p, a, c, k, e, d))
+    sUnpacked1 = str(__unpack(p1, a1, c1, k1, e, d, iteration))
 
-    #print sUnpacked[:200]+'....'+sUnpacked[-100:], len(sUnpacked)
```
Comment (nazegnl) [inline lib/openscrapers/modules/regex.py]: Remove dead code

File: lib/openscrapers/sources_openscrapers/en/extramovies.py [hunk 6]
```diff
@@ -23,15 +23,15 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 '''
 
-
-<<<<<<< Updated upstream
+<< << << < Updated
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/extramovies.py]: Merge artifact this cannot work :)

File: lib/openscrapers/sources_openscrapers/en/extramovies.py [hunk 7]
```diff
@@ -23,15 +23,15 @@
...
+upstream
 
 import traceback
 import urllib
 
 import urlparse
 from openscrapers.modules import cfscrape, cleantitle, log_utils
-=======
+== == == =
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/extramovies.py]: Merge artifact this cannot work :)

File: lib/openscrapers/sources_openscrapers/en/extramovies.py [hunk 8]
```diff
@@ -68,7 +68,9 @@ def random_agent():
     index = random.randrange(len(RAND_UAS))
     return RAND_UAS[index].format(win_ver=random.choice(WIN_VERS), feature=random.choice(FEATURES),
                                   br_ver=random.choice(BR_VERS[index]))
->>>>>>> Stashed changes
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/extramovies.py]: Merge artifact this cannot work :)

File: lib/openscrapers/sources_openscrapers/en/to_be_fixed/iwaatch.py [hunk 9]
```diff
@@ -40,12 +40,11 @@ def __init__(self):
         self.sources = []
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/to_be_fixed/iwaatch.py]: Duplicate with the one in the main folder

File: lib/openscrapers/sources_openscrapers/en/to_be_fixed/azmovie.py [hunk 10]
```diff
@@ -24,7 +24,6 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/to_be_fixed/azmovie.py]: Duplicate

File: lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/cartoonwire.py [hunk 11]
```diff
@@ -22,158 +22,178 @@
...
+
+    def sources(self, url, hostDict, hostprDict):
+        sources = []
+        try:
+            r = client.request(url)
+            try:
+                match = re.compile('var filmId = "(.+?)"').findall(r)
+                for film_id in match:
+
+                    # VIP
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/cartoonwire.py]: Remove commented out code :)

File: lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/seriesfree.py [hunk 12]
```diff
@@ -25,26 +25,26 @@
...
+        self.domains = ['watchseriesfree.to', 'seriesfree.to']
         self.base_link = 'https://seriesfree.to'
         self.search_link = 'https://seriesfree.to/search/%s'
 
     def tvshow(self, imdb, tvdb, tvshowtitle, localtvshowtitle, aliases, year):
         try:
             query = self.search_link % urllib.quote_plus(cleantitle.query(tvshowtitle))
             result = client.request(query)
-            #tvshowtitle = cleantitle.get(tvshowtitle)
+            # tvshowtitle = cleantitle.get(tvshowtitle)
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/seriesfree.py]: Remove commented out code

File: lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/seriesfree.py [hunk 13]
```diff
@@ -83,35 +83,36 @@ def sources(self, url, hostDict, hostprDict):
...
+
                 except:
-                    #traceback.print_exc()
-                    pass           
-                    
-            #log_utils.log('JairoxDebug2: %s' % (str(sources)), log_utils.LOGDEBUG)
+                    # traceback.print_exc()
+                    pass
+
+                    # log_utils.log('JairoxDebug2: %s' % (str(sources)), log_utils.LOGDEBUG)
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/seriesfree.py]: Remove commented out code

File: lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/watchseries.py [hunk 14]
```diff
@@ -15,227 +15,218 @@
...
-			# responses (legit & BS both) are actually very fast: timeout not that important
-			for i in range(4):
-				result = client.request(url, timeout=3)
-				if not result == None: break
-			
-				
-
-			# grab all 'a' that are staged with ep-links formating
-			items = dom_parser.parse_dom(result, 'a', attrs={'itemprop':'url'})
-			#
```
Comment (nazegnl) [inline lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/watchseries.py]: Remove

**Comment by host505** [comment]
@nazegnl I think those scrapers in the to be fixed etc folders are intentionally left there, they are alternative/broken versions of those on the normal folders.
But tbh I don't see a reason bundling them with the script, maybe a new repo should be created for them, for archive reasons, or if anyone wants to fix them?

**Comment by nazegnl** [comment]
@host505 I Think we should keep them to try and fix them later. I think some refactorings in the way the scrapers are setup will enhance results aswell. But first we need to focus on the new CF and its protection :)

**Comment by A4K-Official** [comment]
new cf was already added to OpenScraper courtesy of A4KScrapers for seren

File: lib/openscrapers/modules/log_utils.py (hunk 2 above)
Comment (A4K-Official) [reply lib/openscrapers/modules/log_utils.py]: ?

File: lib/openscrapers/modules/log_utils.py (hunk 3 above)
Comment (A4K-Official) [reply lib/openscrapers/modules/log_utils.py]: ?

File: lib/openscrapers/modules/log_utils.py (hunk 4 above)
Comment (A4K-Official) [reply lib/openscrapers/modules/log_utils.py]: ?

File: lib/openscrapers/modules/regex.py (hunk 5 above)
Comment (A4K-Official) [reply lib/openscrapers/modules/regex.py]: isnt that what this is?

File: lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/watchseries.py (hunk 14 above)
Comment (A4K-Official) [reply lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/watchseries.py]: ?

Comment (A4K-Official) [reply lib/openscrapers/sources_openscrapers/en/to_be_fixed/no_sources/watchseries.py]: oh remove scraper xD lol

**Pull request merged**